### PR TITLE
Avoid compiler error on ESP8266 and ESP32

### PR DESCRIPTION
**Pull request by hb020**
TLDR : Set explicit types for calls to Wire.requestFrom, as the signature was ambiguous.

More detail:

The signatures of Wire.requestFrom that I know of are:

Specific for ESP8266 (https://github.com/esp8266/Arduino/blob/master/libraries/Wire/Wire.cpp):
* size_t TwoWire::requestFrom(uint8_t address, size_t size, bool sendStop)

Specific for ESP32 (https://github.com/espressif/arduino-esp32/blob/master/libraries/Wire/src/Wire.cpp)
* size_t TwoWire::requestFrom(uint16_t address, size_t size, bool sendStop)
* size_t TwoWire::requestFrom(uint8_t address, size_t len, bool sendStop)
* uint8_t TwoWire::requestFrom(uint16_t address, uint8_t len, uint8_t sendStop)
* uint8_t TwoWire::requestFrom(uint16_t address, uint8_t len, bool stopBit)
* uint8_t TwoWire::requestFrom(uint16_t address, uint8_t len)

Specific for ArduinoCore-avr (https://github.com/arduino/ArduinoCore-avr/blob/master/libraries/Wire/src/Wire.cpp):
* uint8_t TwoWire::requestFrom(uint8_t address, uint8_t quantity, uint32_t iaddress, uint8_t isize, uint8_t sendStop)

Common for all three:
* uint8_t TwoWire::requestFrom(uint8_t address, uint8_t quantity, uint8_t sendStop)
* uint8_t TwoWire::requestFrom(uint8_t address, uint8_t quantity)
* uint8_t TwoWire::requestFrom(int address, int quantity)
* uint8_t TwoWire::requestFrom(int address, int quantity, int sendStop)

Calling with (int,int,bool), as originally present, is ambiguous for ESP8266 and ESP32, but not for ArduinoCore-avr. 
A signature that would be unambiguous is (int,int,int) or (uint8_t,uint8_t,uint8_t).
So I went for (int,int,int), as that was the closest to the original code.

Frankly, (uint8_t,size_t,bool) should be the correct one, but avr does not have that as an explicit signature.

In the issue https://github.com/budryerson/TFMini-Plus-I2C/issues/12 I proposed another mapping, but that would be wrong for ArduinoCore-avr.

Not strictly related: 1.7.2 is from 2022, not 2021. I did not correct that.

### CODE REVIEW

**Comment by budryerson** [comment]
Hi.  I know nothing about "push/merge," but I suspect that this issue is keeping me from editing the library.  While I deeply appreciate your gentle corrections and I welcome your collaboration, I'm going to close this issue for now.  In the meantime, can you please tell me more about yourself?  I would like to be able to give you credit for your contributions.
Thanks,
Bud
